tests/integration/test_package_int.py

```diff
@@ -78,6 +78,10 @@ def test_package_flit(initproj, cmd):
 
 @pytest.mark.network
 @pytest.mark.skipif(sys.version_info < (3, 0), reason="poetry is Python 3 only")
+@pytest.mark.xfail(
+    os.name == "nt",
+    reason="https://github.com/tobgu/pyrsistent/issues/88 breaks easy poetry install",
```

Comment:
```suggestion
    reason="https://github.com/tobgu/pyrsistent/issues/88 breaks poetry install",
```

(or am I misunderstanding the message here)